meta-ov/recipes-apps/ovmenu-ng-skripts/files/transfer-xcsoar.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
```

Comment:
Can we do without bash? I think this script is simple enough, and eliminating bash would allow us to cut 1 MB from the image.

Reply:
i am ok without bash.

Reply:
download-igc.sh presently depends on bash. but that thing is anyway in need of a rework.

Reply:
this script will run on sh just fine. converted.